inc/api-extensions.php

```diff
@@ -0,0 +1,30 @@
+<?php
+function hale_extend_wp_api($data, $post, $context) {
+
+    // Add featured image to API
+    $featured_image_id = $data->data['featured_media']; // get featured image id
+    $featured_image_url = wp_get_attachment_image_src( $featured_image_id, 'feature' ); // get url of the feature size
+
```

Comment:
what happens if $featured_image_id is blank

Reply:
If it is blank/null/false, then it will be set to a zero-length string by this code:
```
    if( $featured_image_url ) {
        $data->data['featured_image_url'] = $featured_image_url[0];
    } else {
        $data->data['featured_image_url'] = "";
    }

Reply:
I'm now setting this to false to begin with, then checking the array key exists as above.